packages/web/app/src/pages/organization-new.tsx

```diff
@@ -3,34 +3,31 @@ import { LogOutIcon } from 'lucide-react';
 import { Button } from '@/components/ui/button';
 import { DottedBackground } from '@/components/ui/dotted-background';
 import { Meta } from '@/components/ui/meta';
+import { CreateOrganizationForm } from '@/components/ui/modals/create-organization';
 import { HiveLogo } from '@/components/v2/icon';
-import { CreateOrganizationForm } from '@/components/v2/modals/create-organization';
 import { Link, useRouter } from '@tanstack/react-router';
 
 export function NewOrgPage(): ReactElement {
   const router = useRouter();
+
+  const handleOnClick = (): void => {
+    void router.navigate({
+      to: '/logout',
+    });
+  };
```

Comment:
It's fine to leave it as is, but it does not make sense to turn those small callbacks into dedicated functions, maybe if the name was different (like "onLogoutClick" or something), but still